QuizApp/AppRouter/AppRouter.swift

```diff
@@ -76,4 +76,10 @@ class AppRouter: AppRouterProtocol {
         navigationController.navigationBar.tintColor = .white
     }
 
+    func showQuizResult(numOfCorrectQuestions: Int, numOfQuestions: Int) {
```

Comment:
Instead of passing two int values to QuizResultViewModel, you can create struct Result that will contain those values and pass the Result object